### PR TITLE
15th task 2308

### PR DESCRIPTION
**Pull request by Clasher370**
Schedule notifications.

### CODE REVIEW

File: app/mailers/send_notification_mail_to_user.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# app/mailers/send_notification_mail_to_user.rb
+class SendNotificationMailToUser
+  def self.run
+    User.find(Card.with_ready_date.map(&:user_id).uniq).each do |user|
```
Comment (X1ting) [inline app/mailers/send_notification_mail_to_user.rb]: Why not `.pluck`? 
WDYT, what the problem with `.map`?

File: app/mailers/cards_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# app/mailers/cards_mailer.rb
+class CardsMailer < ApplicationMailer
+  default from: 'notifications@example.com'
+
+  def pending_cards_notification(user)
+    @user = user
```
Comment (X1ting) [inline app/mailers/cards_mailer.rb]: why you assign user to instance variable?

File: app/mailers/cards_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    mail(to: @user.email, subject: 'Пора проверять карты!')
```
Comment (X1ting) [inline app/mailers/cards_mailer.rb]: why not `user.email`?

File: spec/mailers/cards_mailer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe CardsMailer, type: :mailer do
```
Comment (X1ting) [inline spec/mailers/cards_mailer_spec.rb]: are you really need `RSpec.describe`? Maybe `describe` ?

File: app/mailers/send_notification_mail_to_user.rb (hunk 1 above)
Comment (Clasher370) [inline app/mailers/send_notification_mail_to_user.rb]: Maybe .map load useless colums?

**Pull request merged**